.github/workflows/build.yaml

```diff
@@ -47,14 +47,14 @@ jobs:
           west update --narrow -o=--depth=1 -o=--recurse-submodules --path-cache ~/zephyr-thingy91-path-cache
           pip install -r zephyr/scripts/requirements.txt
           pip install -r nrf/scripts/requirements-build.txt
-          west build -b thingy91_nrf9160_ns -p always memfault-asset-tracker
+          west build -b thingy91/nrf9160/ns -p always memfault-asset-tracker
```

Comment:
we'll need to update `ota-deploy.yaml` too i think

Reply:
I got `ota-deploy.yaml` but let me know if I missed something else!

Reply:
Ooh I just realized because we are using sysbuild, the symbol file and `app_update.bin` will have a different path. I'll fix that now

Reply:
Confirmed failure here :D https://github.com/memfault/memfault-asset-tracker/actions/runs/10854721471/job/30125772939

Reply:
Updated. Tested a deployment in a different Memfault project with those filepaths and it works. Double confirming with a manual trigger here: https://github.com/memfault/memfault-asset-tracker/actions/runs/10855350637

Reply:
whoops wrong branch. new run here: https://github.com/memfault/memfault-asset-tracker/actions/runs/10855508979

Reply:
Okay that passed! All good now